modules/ols-creating-lightspeed-custom-resource-file-using-web-console.adoc

```diff
@@ -55,13 +57,13 @@ spec:
     - credentialsSecretRef:
         name: openai-api-keys
       models:
-      - name: models/granite-7b-redhat-lab
+      - name: models/<model_name>
       name: rhelai
       type: rhelai_vllm
-      url: <URL> <1>
+      url: <URL> # <1>
```

Comment:
The `#` isn't hurting anything but FYI is not needed

Reply:
done